class oop.py

```diff
@@ -0,0 +1,79 @@
+class Critter(object):
+    """ The virtual pet """
+    def __init__(self, name, hunger=0, boredom=0):
+        self.name = name
+        self.hunger = hunger
+        self.boredom = boredom
+
+    def __pass_time(self):
+        self.hunger += 1
+        self.boredom += 1
+
+    @property
+    def mood(self):
+        unhappiness = self.hunger + self.boredom
+        if unhappiness < 5:
+            m = 'happy'
+        elif 11 <= unhappiness <= 15:
+            m = 'frustrated'
+        else:
+            m = 'mad'
+        return m
```

Comment:
Can you explain this logic in words?

Reply:
line 17 is 

if unhappiness is 11-15 then m = frustrated

Reply:
Then for what numbers is it "mad"?

Reply:
anything else other than line 15 and 17

Reply:
so 1-5 and 11-15 isn't mad
the rest is

Reply:
Got it, thanks